test/contracts/async-call-child/elrond.json

```diff
@@ -0,0 +1,6 @@
+{
+    "language": "clang",
+    "source_files": [
+        "/home/camil.bancioiu/Work/Elrond/arwen-wasm-vm/test/contracts/async-call-child/async-call-child.c"
```

Comment:
is this link good ? i think this should be relative path

Reply:
Not good. Fixed.